x-pack/plugins/apm/ftr_e2e/cypress/integration/power_user/feature_flag/comparison.ts

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { synthtrace } from '../../../../synthtrace';
+import { opbeans } from '../../../fixtures/synthtrace/opbeans';
+
+const settingsPath = '/app/management/kibana/settings';
+const apmBasePath = '/app/apm';
+
+const start = '2021-10-10T00:00:00.000Z';
+const end = '2021-10-10T00:15:00.000Z';
+describe('Comparison feature flag', () => {
+  const comparisonToggle =
+    '[data-test-subj="advancedSetting-editField-observability:enableComparisonByDefault"]';
+
+  before(async () => {
+    await synthtrace.index(
+      opbeans({
+        from: new Date(start).getTime(),
+        to: new Date(end).getTime(),
+      })
+    );
+  });
+
+  after(async () => {
+    await synthtrace.clean();
+  });
+
+  beforeEach(() => {
+    cy.loginAsPowerUser();
+  });
+
+  describe('when comparison feature is enabled', () => {
+    it('shows the flag as enabled in kibana advanced settings', () => {
+      cy.visit(settingsPath);
+
+      cy.get(comparisonToggle)
+        .should('have.attr', 'aria-checked')
+        .and('equal', 'true');
+    });
+
+    it('shows the comparison feature enabled in services overview', () => {
+      cy.visit(`${apmBasePath}/services`);
```

Comment:
Are there any advantages in abstracting the base url? I think it reduces readability and makes it impossible to find the path when searching across the entire code base:

```suggestion
      cy.visit(`/app/apm/services`);
```

Reply:
Other than reusing the apm base path there are no other advantages (imo) so in favour of code readability it worth changing it